faunadb-scala/src/main/scala/faunadb/values/Codec.scala

```diff
@@ -145,11 +198,12 @@ object Decoder {
       }
   }
 
-  implicit object ByteArrayDecoder extends Decoder[Array[Byte]] {
+  implicit def EitherDecoder[A, B](implicit a: Decoder[A], b: Decoder[B], tagA: ClassTag[A], tagB: ClassTag[B]): Decoder[Either[A, B]] = new Decoder[Either[A, B]] {
```

Comment:
this is to get the name of classes `A` and `B` and format the error message, I tried to use `classOf[A]` but it only expect a static type